### PR TITLE
tweak the readme language a bit

### PR DESCRIPTION
**Pull request by nicks**
Hello @dbentley, @ellenkorbes,

Please review the following commits I made in branch nicks/readme:

397727004e0d2926b09ee63680a64896748d703c (2020-04-27 12:35:46 -0400)
tweak the readme language a bit

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

**Comment by nicks** [comment]
Trying to update the headline language a bit to work in the category name. If this looks good, I'll work it in a few other places too

File: README.md [hunk 1]
```diff
@@ -5,11 +5,12 @@
 [![Build Status](https://circleci.com/gh/windmilleng/tilt/tree/master.svg?style=shield)](https://circleci.com/gh/windmilleng/tilt)
 [![GoDoc](https://godoc.org/github.com/windmilleng/tilt?status.svg)](https://godoc.org/github.com/windmilleng/tilt)
 
-Local Kubernetes development with no stress.
+Kubernetes for Prod, Tilt for Dev
 
-[Tilt](https://tilt.dev) helps you develop your microservices locally.
-Run `tilt up` to start working on your services in a complete dev environment
-configured for your team.
+Modern apps are made of multiple services: a database, an API server, a web bundler.
```
Comment (ellenkorbes) [inline README.md:10]: Can we make this more generic? `a frontend, a backend, a database`?

Comment (nicks) [reply README.md:10]: ya, i struggled with this a bit.

I disagree that "a frontend, a backend, a database" evokes a neutral, generic architecture. It evokes a very particular architecture. So I wanted that seemed like it applied even if you were doing something like Jamstack or Celery tasks. wdyt?

File: README.md [hunk 2]
```diff
@@ -5,11 +5,12 @@
...
+[Tilt](https://tilt.dev) powers multi-service development.
+Run `tilt up` to start working in a complete dev environment configured
+for your team.
```
Comment (ellenkorbes) [inline README.md:13]: `s/to start working/to work`

File: README.md [hunk 3]
```diff
@@ -5,11 +5,12 @@
...
 
 Tilt watches your files for edits, automatically builds your container images,
 and applies any changes to bring your environment
```
Comment (ellenkorbes) [inline README.md:16]: The way this is phrased makes it seem like three separate points, not like one continuous operation. Rephrase?

File: README.md (hunk 1 above)
Comment (ellenkorbes) [reply README.md:10]: I think it doesn't work.

`Modern apps are made of a myriad of different services in constant communication.` ?

Comment (nicks) [reply README.md:10]: ooh, i like the idea of leaning into the idea of escalating complexity.

How about:

```
Modern apps are made of too many services. They're everywhere and in constant communication. 

[Tilt](https://tilt.dev) powers multi-service development so you can keep an eye on them.
```

too much?

File: README.md (hunk 3 above)
Comment (nicks) [reply README.md]: ya, i agree. borrowed some of the language from the example guides, how's this?

> Tilt automates all the steps from a code change to a new process: watching files, building container images, and bringing your environment up-to-date.

Comment (ellenkorbes) [reply README.md]: Perfect!

File: README.md (hunk 1 above)
Comment (ellenkorbes) [reply README.md]: `so you can keep an eye on them` feels like passive voice to me.

I'd go for something like `and keeps them in line.`, `and makes sure they behave!` Something like that?

The gist of the edit is that I don't want "to be able to do something if I so please." A more active sentence construction works best for copy editing. (Think movie-trailer voice.)

Comment (nicks) [reply README.md]: haha, i :green_heart: both of those suggestions, i went with the more sassy one!

**Pull request merged**